index.html

```diff
@@ -13,9 +14,9 @@
 			<div>
 				<ul>
 
-					<li>about me</li>
```

Comment:
These lines go off the image and merge with the gradient at the side when I shrink the image. Perhaps fix this by making it stay in one place?

Reply:
Yes, I noted this in my comment on the pull request: I'll resolve it next.

Reply:
Oh, I had misunderstood what you meant.  Yes, the text would be better fixed in an ideal world. I'll see if I have time!